fn:error() instance of xs:integer
